eng/Versions.props

```diff
@@ -97,12 +101,12 @@
     <MicrosoftNetCompilersToolsetPackageVersion>3.9.0-2.20573.10</MicrosoftNetCompilersToolsetPackageVersion>
     <MicrosoftNetRoslynDiagnosticsPackageVersion>2.6.3</MicrosoftNetRoslynDiagnosticsPackageVersion>
     <MicrosoftServiceHubFrameworkPackageVersion>2.7.313-preview</MicrosoftServiceHubFrameworkPackageVersion>
-    <MicrosoftVisualStudioCoreUtilityPackageVersion>16.8.272</MicrosoftVisualStudioCoreUtilityPackageVersion>
+    <MicrosoftVisualStudioCoreUtilityPackageVersion>16.10.8</MicrosoftVisualStudioCoreUtilityPackageVersion>
```

Comment:
📝 This update is caused by `Tooling_HtmlEditorPackageVersion`. If it's not acceptable to update now for the whole solution, I can separate the dependencies used in shipping code from the dependencies used in tests.

Reply:
Could this cause problems because the repo doesn't have long-lived servicing branches e.g. theres's no release/vs16.8❔ @NTaylorMullen are the tags enough❔

Reply:
Even if we had longer lived branches why would this cause problems? Not sure I follow

Reply:
With long-lived servicing branches or enough tags to create branches when needed, there's no problem. However the changes in this file imply supporting only 16.10 going forward in the master branch. Are the existing branches and / or tags to create new branches sufficient to cover supporting 16.X where `X < 10`❔

Reply:
Yup they're sufficient

Reply:
@dougbu Servicing would never occur from the master/main branch. We would either use a dedicated servicing branch, or create one at the time it's needed at a commit marked by an earlier tag. The main concern with this change is not the impact on releases, but rather the fact that it will no longer be able to <kbd>F5</kbd>-debug the main branch in 16.9.x, and external users don't yet have access to 16.10 preview builds.

We have to make a decision regarding the tradeoff between ability to debug the latest commits in the main branch and the complexity involved in using one set of dependences for production code and a different set of dependencies for tests.

Not that all users will still be able to debug _tests_ with this change.